lrpar/src/lib/lex.rs

```diff
@@ -62,6 +61,14 @@ pub trait Lexer<StorageT: Hash + PrimInt + Unsigned> {
     ///
     /// If the span exceeds the known input.
     fn span_lines_str(&self, span: Span) -> &str;
+
+    /// Return `((start line, start column char), (end line, end column char))` for `span`. Note
```

Comment:
What is `start column char`. I thought this would just return the line and column, e.g. `4, 5`. Does this have to do with the fact that we are returning chars and not bytes? Either way `start column char` is a bit confusing as it sounds like this returns a character instead of an index. Maybe `start column char index` would be better, if a bit long though.

Reply:
Think of it as `(start, end)` where both `start` and `end` have `(line, column char)`. So, yes, it's because we return chars not bytes. I suppose we could say `((start line, start column char off)), ...)` or similar?

Reply:
I think just shortening it to `start line, start column` is probably fine, especially with the note further down explaining that we count chars not bytes.

Reply:
Fixed in https://github.com/softdevteam/grmtools/pull/154/commits/0b522f3a06dedcaf711cb8d7a644294ea6e04cef.